### PR TITLE
feat: add support for url wildcards in the settings

### PR DESCRIPTION
**Pull request by Pacos**
This change allows users to use wildcards in the URLs defined for staging & prod in the parameters.

It also includes minor fixes in the README following the repository's renaming.

### CODE REVIEW

File: src/contentScript.js [hunk 1]
```diff
@@ -45,17 +45,19 @@ function highlightDags(dags, style) {
...
+        if (isUrlMatched) {
             document
                 .getElementsByClassName("navbar")[0]
                 .setAttribute("style", "background-color: " + data.color + "!important");
             document
                 .getElementsByClassName("active")[0]
                 .children[0].setAttribute("style", "background-color: #00000020");
-            break;
         }
-    }
```
Comment (llandr20) [inline src/contentScript.js]: Is there an extra indentation here ?

**Review by llandr20 — COMMENTED** [review]
It works perfectly on my computer.
Should we still increase the version ?

**Comment by Pacos** [comment]
> It works perfectly on my computer. Should we still increase the version ?

Did you try it on Chrome and/or Firefox ? I was only able to test it on Firefox for now

File: src/contentScript.js (hunk 1 above)
Comment (Pacos) [reply src/contentScript.js]: Good catch ! I fixed that

**Comment by rinzool** [comment]
> > It works perfectly on my computer. Should we still increase the version ?
> 
> Did you try it on Chrome and/or Firefox ? I was only able to test it on Firefox for now

Works perfectly on chrome, thanks :+1:

**Review by rinzool — DISMISSED** [review]

**Review by rinzool — APPROVED** [review]

**Pull request merged**